### DISCUSSION TITLE
Capitalize appname

### DISCUSSION THREAD

**Discussion by tct123** [flet-dev/flet#4258] [General]
How exactly do I get the app name to be capitalized in the Android launcher in python code? @ndonkoHenri 
I know, I allready asked that on discord, but I didnt got a good answer.

**Comment by tct123** [comment]
The actual behaviour is, that the name is only written in small letters.
![Screenshot_20241025-145053~2](https://github.com/user-attachments/assets/3cdd1dda-a048-4a11-8611-ad90816fa1e7)

**Reply by FeodorFitsner** [reply]
Use `--product` option: `flet build --product "My Flet App"` to set display name.

**Reply by ndonkoHenri** [reply]
You can also [define](https://flet.dev/blog/pyproject-toml-support-for-flet-build-command#quick-start) it in your `pyproject.toml`
